libs/api/database/src/lib/content/schemas/blogs-content.schema.ts

```diff
@@ -25,6 +25,10 @@ export class BlogsContentDocument extends Document implements BlogsContentModel
     };
 
     readonly kind: ContentKind;
+
+    @Prop({type: [{type: MongooseSchema.Types.ObjectId, ref: 'TagsDocument'}] })
+    tags: TagsModel[];
```

Comment:
The syntax is a little wacky here, but it should be correct.

Reply:
looks good to me